### PR TITLE
Add some fix for azsecpack test case

### PR DESCRIPTION
**Pull request by lubaihua33**

### CODE REVIEW

File: microsoft/testsuites/vm_extensions/azsecpack.py [hunk 1]
```diff
@@ -104,17 +118,26 @@ def verify_azsecpack(self, node: Node, log: Logger, result: TestResult) -> None:
...
+        # Install agent extension
+        try:
+            self._install_monitor_agent_extension(node)
+            self._install_security_agent_extension(node, log)
+        except HttpResponseError as identifier:
+            if any(
+                s in str(identifier)
+                for s in ["OS is not supported", "Unsupported operating system"]
+            ):
+                raise SkippedException(f"unsupported Linux distro: {node.os}")
```
Comment (squirrelsc) [inline microsoft/testsuites/vm_extensions/azsecpack.py]: Raise like SkipppedException(UnsupportedOsException(...))

File: microsoft/testsuites/vm_extensions/azsecpack.py [hunk 2]
```diff
@@ -282,29 +310,40 @@ def _check_journalctl_logs(self, node: Node, log: Logger) -> None:
...
+        for distro in supported_major_versions:
+            if type(node.os) == distro:
+                version_list = supported_major_versions.get(distro)
+                if (
+                    version_list is None
+                    or node.os.information.version.major not in version_list
+                ):
+                    raise SkippedException(
+                        UnsupportedDistroException(
+                            node.os, "AzSecPack doesn't support this Distor version."
```
Comment (LiliDeng) [inline microsoft/testsuites/vm_extensions/azsecpack.py]: Distro

File: microsoft/testsuites/vm_extensions/azsecpack.py (hunk 1 above)
Comment (lubaihua33) [reply microsoft/testsuites/vm_extensions/azsecpack.py]: Done

File: microsoft/testsuites/vm_extensions/azsecpack.py (hunk 2 above)
Comment (lubaihua33) [reply microsoft/testsuites/vm_extensions/azsecpack.py]: Done

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**